sapi/cli/php_cli.c

```diff
@@ -558,16 +558,19 @@ static void cli_register_file_handles(void)
 	php_stream_to_zval(s_err, &ec.value);
 
 	Z_CONSTANT_FLAGS(ic.value) = 0;
-	ic.name = zend_string_init_interned("STDIN", sizeof("STDIN")-1, 0);
-	zend_register_constant(&ic);
+	zend_string *stdin_name = zend_string_init_interned("STDIN", sizeof("STDIN")-1, 0);
+	zend_register_constant(stdin_name, &ic);
+	zend_string_release(stdin_name);
```

Comment:
You may use ``zend_register_internal_constant()``.

Reply:
Thanks for the suggestion!